app/views/content/non-uk-teachers/_why-teach.html.erb

```diff
@@ -1,11 +1,9 @@
-<section class="why-teach" aria-label="Why teach?">
+<section class="why-teach" aria-label="Join a world class education system">
```

Comment:
@garethwilliamsdesign @gemmadallmandfe I think this would be better with the H2 as Why teach in England? and we get rid of the Join a world class education system. But if you disagree and want to leave it I think world-class should be hyphenated.

Reply:
I'm happy with your suggestion if @gemmadallmandfe is, too?

Reply:
Yes - happy with @MaxineCl's suggestion!